js/main.js

```diff
@@ -69,7 +69,7 @@
             clearChildren(username);
             var a = document.createElement("a");
             a.setAttribute("target", "blank");
-            a.href = "https://github.com/" + user;
+            a.href = "https://github.com/" + (user.startsWith("@") ? user.slice(1) : user);
```

Comment:
Why? This is a trivial task that can be done with a simple native method, and involves literal strings - nothing that we can't do without regexes. I hate to leave you off with a rant, but using regexes for tasks like these will result in poor performance and is definitely why we can't have nice things: http://h2co3.org/blog/index.php/2014/07/28/this-is-why-we-cant-have-nice-things/